CONTRIBUTING.md

```diff
@@ -7,19 +7,13 @@
 Once the repo has been cloned, install Lerna:
 
 ```bash
-npm install -g lerna
+yarn global add lerna
 ```
 
-Install dependencies in the root directory:
+Install all package dependencies:
 
 ```bash
-npm install
-```
-
-Install dependencies for packages within the project:
-
-```bash
-lerna bootstrap
```

Comment:
Can you add a new step here - `lerna run prepare`.

That command will largely take the place of `lerna bootstrap` with yarn

Reply:
Updated - Do we perhaps want to see if we can setup a githook for post-install to just run this?

Reply:
...in a different PR

Reply:
Yea let's do any of that as a follow up. I'm not convinced yet that we really need to add additional complications, but if we do I think it will become readily apparent.